.github/workflows/helm.yml

```diff
@@ -0,0 +1,41 @@
+name: Helm CI
+on:
+  push:
+    paths:
+      - 'chart/**'
+      - '!chart/README.md'
+      - '!chart/.helmignore'
+  pull_request:
+    paths:
+      - 'chart/**'
+      - '!chart/README.md'
+      - '!chart/.helmignore'
+jobs:
+  lint:
+    if: github.ref != 'refs/heads/staging' && github.ref != 'refs/heads/production'
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.lint.outputs.changed }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Lint chart
+        id: lint
+        uses: helm/chart-testing-action@v1.0.0
+        with:
+          command: lint
+          config: .ct.yaml
+  test:
+    needs: lint
+    if: github.ref != 'refs/heads/staging' && github.ref != 'refs/heads/production' && needs.lint.outputs.changed == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.0.0
+      - name: Install chart
+        uses: helm/chart-testing-action@v1.0.0
+        with:
+          command: install
+          config: .ct.yaml
```

Comment:
I am a huge proponent of pre-commit. Would you be opposed to me adding a minimal pre-commit config to the repo as well? (this would ensure the aforementioned is a non-issue, and could ensure things like sign offs are done)

Reply:
I think this is up to @mattfarina, @tegioz and @cynthia-sg as maintainers of this project. I don't have an opinion. 

Though this can also be caught in CI, and maybe it should? I haven't checked if the [GitHub Superlinter Action](https://github.com/github/super-linter) does newline at EOF checks, but may be worth looking into.

Reply:
I think it's better if all this kind of checks are centralized in CI.